Caltech-IPAC/firefly
# FIREFLY-1086: Click to Action

#### [FIREFLY-1086](https://jira.ipac.caltech.edu/browse/FIREFLY-1086): Click to Action
 - defined data structure to support actions
 - made UI to load searches
 - made a Search Refinement tool

#### Testing
 - https://fireflydev.ipac.caltech.edu/firefly-1086-click/firefly/
 - https://firefly-1086-click.irsakudev.ipac.caltech.edu/irsaviewer/
 - Load an image or a HiPS
   - Select an area
   - Click on searches to see options
 - Do the same thing but this time click on search and choose "search refinement tool"
 - This time use the IrsaViewer url. Load a hips, select an select and 1 degree area, and choose search sofia
